SR2019/2018-09-27-kickstart-announcement.md

```diff
@@ -0,0 +1,16 @@
+---
+to: All Teams
+subject: SR2019 Registration and Kickstart Announcement
+---
+
+After a year's hiatus during which we've worked to ensure that Student Robotics will be more scalable for the future, we're pleased to announce the continuation of the annual competition. As always, we start this competition period with a Kickstart event. Here, we will announce the game, and show you how to use the kit.
+
+In order to allow the core team of volunteers to focus on delivering a high quality programme while requiring far fewer resources than before, this year's competition will be slightly smaller than previous years. We expect to be able to increase the size of the competition again in the future.
+
+Kickstart will be on **Saturday 10th November**. This time, we're planning on running 2 events. 1 at the **University of Southampton**, and the second in **London**. The exact locations will be confirmed in the next couple of weeks, however the events will run from around 10am, and finish some time before 5pm. This information will be reflected on our website soon. If you are unable to make either of these, please let us know!
```

Comment:
Can we frame this in such a way that it's clear that:
- these events are invite-only, and
- that we'll tell teams which venue to go to

The latter obviously assuming that that is our intent.

Reply:
I've added a field to the signup form saying "Desired kickstart location", which I think covers the latter point. 

I think the invite-only component is obvious, seeing as only certain people will be getting all the details anyway.

Reply:
ok, for clarity this was particularly in response to:

>  If you are unable to make either of these, please let us know!

which feels like it implies that this email is an invite.

Reply:
I'll remove that line, and capture it in the form instead